apps/filebrowser/src/filebrowser/templates/fb_components.mako

```diff
@@ -58,9 +59,15 @@ else:
         %endif
         <li>
           <ul id="editBreadcrumb" class="hue-breadcrumbs editable-breadcrumbs" data-bind="foreach: breadcrumbs" style="padding-right:40px; padding-top: 12px" title="${_('Edit path')}">
-            <li data-bind="visible: label.slice(-1) == '/'">
-              <a data-bind="click: show, attr: {'href': '${url('filebrowser:filebrowser.views.view', path=urlencode(''))}' + url}"><span class="divider" data-bind="text: label"></span></a>
-            </li>
+            %if RAZ.IS_ENABLED.get():
+              <li data-bind="visible: label.slice(-1) == '/'">
+                <a><span class="divider" data-bind="text: label"></span></a>
```

Comment:
Empty anchor. Do we even need to show the <li> when Raz is enabled?

Reply:
then how we will show the path of a current folder?